src/machine.rs

```diff
@@ -1392,11 +1392,7 @@ impl OlmMachine {
     /// Shared helper for `import_exported_room_keys` and `import_room_keys`.
     ///
     /// Wraps the progress listener in a Rust closure and runs
-    /// `Store::import_exported_room_keys`
-    ///
-    /// Items inside `exported_room_keys` will be invalidated by this method. Be
-    /// careful not to use the `ExportedRoomKey`s after this method has been
-    /// called.
```

Comment:
this method isn't actually called by javascript. So this is just standard rust semantics, which don't need explicit documentation.